case-lib/hijack.sh

```diff
@@ -101,8 +101,8 @@ function func_exit_handler()
 
     fi
 
-    if is_ipc4 && is_firmware_file_zephyr; then
-	local mtraceBin; mtraceBin=mtrace-reader.py
+    if [ "$SOF_LOGGING" != 'none' ] && is_ipc4 && is_firmware_file_zephyr; then
+        local mtraceBin; mtraceBin=mtrace-reader.py
```

Comment:
Whitespace change: Good thing this isn't Python. Rest of the change isn't an issue so I can still approve.

Reply:
I hate unrelated whitespace fixes because they break git cherry-pick, merge, blame etc. but in this case the change is so close that git would conflict anyway.